platform/templates/secrets/ldap.yaml

```diff
@@ -1,13 +1,11 @@
-{{- if .Values.ldap.enabled }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: ldap
 type: Opaque
 data:
-  host: bGRhcA== #ldap
-  base_dn: ZGM9ZXhhbXBsZSxkYz1vcmc= #dc=example,dc=org
-  user: YWRtaW51c2VybmFtZT1hZG1pbg== #adminusername=admin
-  password: cGFzc3dvcmQ9YWRtaW4= #password=admin
-  environment_ou: dGVzdA== #test
-{{- end }}
+  host: bGRhcC1vcGVubGRhcA== # ldap-openldap
+  base_dn: ZGM9ZXhhbXBsZSxkYz1vcmc= # dc=example,dc=org
+  user: Y249YWRtaW4= # cn=admin
+  password: VGhpc0lzNG40ZG1pblA0c3N3b3Jk # ThisIs4n4dminP4ssword
```

Comment:
We should probably change this back to `admin`, since `admin/admin` is a pretty standard default for stuff like this.